sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -26,3 +26,23 @@ export interface StudentApplicationDTO {
   award: string;
   status: string;
 }
+
+export interface LocationsApplicationDTO {
```

Comment:
IMO, we should try to call it something more directly related to the "Program Info" instead of application.